lib/api.js

```diff
@@ -46,58 +46,72 @@ class ApiRequest {
   }
 
   /**
-   * @param {Object} options
-   * @description Append URI.
+   * Append URI.
+   * @param {Object} options - The request options.
+   * @param {Number} host - The request host.
    */
-  static appendURI (options) {
-    options.uri = `${API_HOST}${options.endpoint}`
+  static appendURI (options, host) {
```

Comment:
why not send endpoint instead of options ?

Reply:
ok, will send endpoint

Reply:
hmmm no, it's setting uri property to options object